addons/intelitems/functions/fnc_addIntel.sqf

```diff
@@ -33,7 +33,7 @@ if (
 
 // Add the magazine item to the unit's inventory and get its id
 private _magazinesBefore = [_unit, _item] call CBA_fnc_getMagazineIndex;
-_unit addMagazine _item;
+_unit addMagazine [_item, 2];
```

Comment:
```suggestion
_unit addMagazine [_item, 1];
```

Reply:
i dont now why i used 2 there... i must be sleepy ^^